docs/scribble/document/util/register_map/tables.py

```diff
@@ -5,18 +5,22 @@
 from scribble import table
 from .models import Register, RegisterField, RegisterMap
 
-
 def get_register_map_table(
-    register_map: RegisterMap, caption: str, reference_id: str
+    register_map: RegisterMap, title: str="", reference_id: str ="", caption: str = ""
```

Comment:
Can you make this consistent with the underlying constructor for `Table` and have all these default to `None` instead of the empty string? See https://github.com/sifive/scribble/blob/a1b231242d0c7d1434131bed814f63d120b939cd/scribble/table.py#L136-L144